### PR TITLE
avoid KeyError in /rest/v1/packageimporttransaction/ api

### PR DESCRIPTION
**Pull request by RichardXuan**
Fixes OLCS-357

### CODE REVIEW

File: openlcs/packages/mixins.py [hunk 1]
```diff
@@ -101,11 +101,12 @@ def __init__(self):
...
     @staticmethod
     def create_component(component_data):
-        summary_license = component_data.pop('summary_license')
+        summary_license = "" if component_data.get('summary_license') is None\
+            else component_data['summary_license']
         defaults = {"summary_license": summary_license}
         if "is_source" in component_data:
             defaults.update({
-                'is_source': component_data.pop("is_source")
+                'is_source': component_data.get("is_source")
```
Comment (hanchuntao) [inline openlcs/packages/mixins.py:109]: Why should change it, I checked the `Component` design, the field exist, and check the log, it also exists
```
{'uuid': '8757d046-94d0-4647-8220-fcca3d22f2d6', 'type': 'GOLANG', 'name': 'sigs.k8s.io/yaml', 'version': 'v1.2.0', 'release': '', 'summary_license': '', 'arch': '', 'is_source': True, 'synced': False} ```

Comment (RichardXuan) [reply openlcs/packages/mixins.py:109]: The reason why change it is  packageimporttransaction api have a for loop(https://github.com/RedHatProductSecurity/openlcs/blob/502d20af64642d096ad3bf8a1bcd1be412593e1a/openlcs/packages/views.py#L497).
And component dict may used multiple times.If a field pop, next time call `create_component` will cause a `KeyError `.

Comment (hanchuntao) [reply openlcs/packages/mixins.py:109]: OK, got it, thanks for your explanation

**Comment by hanchuntao** [comment]
LGTM

**Pull request merged**